appg.adoc

```diff
@@ -93,3 +93,6 @@ Re-do several changes which had previously been made in an earlier draft of vers
 .20 January 2016
 . link:$$http://cf-trac.llnl.gov/trac/ticket/67$$[Ticket 67], remove
 deprecation of "missing_value" from <<attribute-appendix>>.
+
+.29 February 2016
+In Appendix A (Attributes), updated Type and Use values for some attributes and added "special purpose" value. In Appendix H (Annotated Examples of Descrete Geometries), updated coordinate values for the variables in some examples to correct omissions.
```

Comment:
Please could you:
1. Add a link to the trac ticket (see previous entry for ticket 67 for an example).
2. Use cross-references to refer to the modified appendices, i.e. `<<attribute-appendix>>` and `<<appendix-examples-discrete-geometries>>`.